src/Symfony/Component/HttpFoundation/composer.json

```diff
@@ -16,25 +16,23 @@
         }
     ],
     "require": {
-        "php": ">=8.2",
+        "php": ">=8.4",
         "symfony/deprecation-contracts": "^2.5|^3.0",
-        "symfony/polyfill-mbstring": "~1.1",
-        "symfony/polyfill-php83": "^1.27"
+        "symfony/polyfill-mbstring": "^1.1"
     },
     "require-dev": {
         "doctrine/dbal": "^3.6|^4",
         "predis/predis": "^1.1|^2.0",
-        "symfony/cache": "^6.4.12|^7.1.5|^8.0",
-        "symfony/clock": "^6.4|^7.0|^8.0",
-        "symfony/dependency-injection": "^6.4|^7.0|^8.0",
-        "symfony/http-kernel": "^6.4|^7.0|^8.0",
-        "symfony/mime": "^6.4|^7.0|^8.0",
-        "symfony/expression-language": "^6.4|^7.0|^8.0",
-        "symfony/rate-limiter": "^6.4|^7.0|^8.0"
+        "symfony/cache": "^7.4|^8.0",
```

Comment:
Is there a reason why we bump dependencies only in some of the packages?

Reply:
because this is required to fix low-deps
but in the end I'll bump everything yes